react/lib/util/number.ts

```diff
@@ -0,0 +1,12 @@
+import BigNumber from 'bignumber.js';
+
+export const zero = new BigNumber(0);
+
+export const resolveNumber = (value: BigNumber.Value) => {
+  return new BigNumber(value);
+};
+
+export const isLessThanZero = (value: BigNumber.Value) => {
```

Comment:
This function is implemented backwards. It's named `isLessThanZero` but it checks if `zeroIsLessThan`